oidc/client.go

```diff
@@ -66,11 +66,16 @@ type ClientConfig struct {
 }
 
 func NewClient(cfg ClientConfig) (*Client, error) {
+	ru, err := phttp.ParseNonEmptyURL(cfg.RedirectURL)
```

Comment:
@sym3tri My use case here is the kube-apiserver acts as a client, but it doesn't handle login or cause to generate any tokens, it just verifies the token.